components/mpas-albany-landice/bld/build-namelist

```diff
@@ -540,7 +553,13 @@ add_default($nl, 'config_max_water_fraction');
 #################################
 
 if ($MALI_PROGNOSTIC_MODE eq 'PROGNOSTIC') {
-    add_default($nl, 'config_basal_mass_bal_float');
+    if ($OCN_GLC_ISMF_COUPLING eq 'tf') {
+        add_default($nl, 'config_basal_mass_bal_float');
```

Comment:
This means it's set to 'none', right? Will that be confusing for users?

Reply:
Good catch - that's a mistake!